scripts/repl.js

```diff
@@ -284,7 +371,34 @@
   /*
    * Initialize the REPL
    */
-  var repl = new REPL();
+  var initialQuery = UriUtils.parseQuery();
+  if (initialQuery && initialQuery.short) {
```

Comment:
Meaning it should leave the url at /repl/ until they hit save, and only support linking with short urls?